.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "arrowParens": "avoid",
+  "printWidth": 140
```

Comment:
Do I smell an ultra wide monitor here? Mind making this 110, only because I like tiling my windows next to each other and sadly don't have an ultra wide.
```suggestion
  "printWidth": 110
```

Reply:
Ah sorry yeah 2K monitor haha. I'll change and recommit it all.

Reply:
Funnily enough, this didn't change any code.